compact_activity_snapshot.proto

```diff
@@ -48,12 +48,12 @@ message Backend {
     WAIT_EVENT_LWLOCK_WAL_BUF_MAPPING_LOCK = 107; // WALBufMappingLock
     WAIT_EVENT_LWLOCK_WAL_WRITE_LOCK = 108; // WALWriteLock
     WAIT_EVENT_LWLOCK_CONTROL_FILE_LOCK = 109; // ControlFileLock
-    WAIT_EVENT_LWLOCK_CHECKPOINT_LOCK = 110; // CheckpointLock
-    WAIT_EVENT_LWLOCK_C_LOG_CONTROL_LOCK = 111; // CLogControlLock
-    WAIT_EVENT_LWLOCK_SUBTRANS_CONTROL_LOCK = 112; // SubtransControlLock
+    WAIT_EVENT_LWLOCK_CHECKPOINT_LOCK = 110; // CheckpointLock (removed in 14)
+    WAIT_EVENT_LWLOCK_C_LOG_CONTROL_LOCK = 111; // CLogControlLock, XactSLRULock
```

Comment:
[note for reviewer] related commits for the rename:
https://github.com/postgres/postgres/commit/5da14938f7bfb96b648ee3c47e7ea2afca5bcc4a
https://github.com/postgres/postgres/commit/36ac359d3621578cefc2156a3917024cdd3b1829